CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ IMPROVEMENTS:
  * ui: Moved client status, draining, and eligibility fields into single state column [[GH-5789](https://github.com/hashicorp/nomad/pull/5789)]
  * ui: Added buttons to copy client and allocation UUIDs [[GH-5926](https://github.com/hashicorp/nomad/pull/5926)]
  * ui: Added page titles [[GH-5924](https://github.com/hashicorp/nomad/pull/5924)]
```

Comment:
Within each subsection (e.g. `ui`) the lines should be in line length order rather than in chronological order.

Reply:
Fixed, thank you!